src/app/WriteHandler.cpp

```diff
@@ -118,7 +118,13 @@ CHIP_ERROR WriteHandler::OnMessageReceived(Messaging::ExchangeContext * apExchan
                        "OnMessageReceived should not be called on GroupExchangeContext");
     if (!aPayloadHeader.HasMessageType(Protocols::InteractionModel::MsgType::WriteRequest))
     {
+        if (aPayloadHeader.HasMessageType(Protocols::InteractionModel::MsgType::StatusResponse))
+        {
+            CHIP_ERROR statusError = CHIP_NO_ERROR;
+            StatusResponse::ProcessStatusResponse(std::move(aPayload), statusError);
```

Comment:
What's the point of this block?  `statusError` never escapes....  If this is just for the logging, please document that.

Reply:
yes, it is for logging purpose!